source/preface.tex

```diff
@@ -90,92 +80,38 @@ \chapter{Foreword}
 \item support for an initializer statement in range-based for loops
 \item support for default member initializers for bit-fields
 \item support for parenthesized aggregate initialization
-\item extended support for lambda expressions, including
-\begin{itemize}
-  \item explicit \grammarterm{template-head}s in generic lambdas
-  \item default construction and assignment of stateless closure types
-  \item lambda expressions in unevaluated operands
-  \item pack expansion of lambda \grammarterm{init-capture}s
-\end{itemize}
-\item generalized support for structured bindings
+\item extensions to lambda expressions
+\item extensions to structured bindings
 \item support for inline namespaces in nested namespace definitions
 \item support for conditionally-explicit member functions
-\item extended support for class template argument deduction to cover
-aggregate initialization and alias templates
+\item extensions to class template argument deduction
 \item reduced cases in which \keyword{typename} is required
 \item support for calling an undeclared \grammarterm{template-id}
 via argument-dependent name lookup
-\item relaxed access checking rules in template specialization declarations
 \item revised memory model
-\item expanded cases in which returned or thrown variables are implicitly moved
 \item extended support for variadic macros with \mname{VA_OPT}
 % library changes
 \item feature test macros and \libheaderref{version} header
-\item restricted valid uses of standard library functions and function templates
 \item addition of ranges and \libheaderref{ranges} header
 \item addition of calendar and time zone support
 \item addition of text formatting library and \libheaderref{format} header
-\item addition of synchronization facilities for
-waiting, notifications, semaphores, latches, and barriers, and
-\libheaderref{barrier}, \libheaderref{latch} and \libheaderref{semaphore} headers
+\item addition of \libheaderref{barrier}, \libheaderref{latch} and \libheaderref{semaphore} headers
```

Comment:
this needs an Oxford comma, I believe

Reply:
Done, and squashed a little more of the more-minor content out of this list.